MSAL/src/native_auth/controllers/reset_password/MSALNativeAuthResetPasswordController.swift

```diff
@@ -184,7 +184,7 @@ final class MSALNativeAuthResetPasswordController: MSALNativeAuthBaseController,
             stopTelemetryEvent(event, context: context)
 
             return .codeRequired(
-                newState: ResetPasswordCodeRequiredState(controller: self, flowToken: challengeToken),
+                newState: ResetPasswordCodeRequiredState(controller: self, continuationToken: challengeToken),
```

Comment:
For consistency, can you please rename also the `challengeToken` to continuationToken? Same in the MSALNativeAuthResetPasswordChallengeValidatedResponse.success type